pmultiqc/modules/quantms/quantms.py

```
@@ -80,167 +80,201 @@ def __init__(self):
-                        ''',
-                plot=self.protein_quantification_table_html
-            )
+        else:
+            self.enable_exp = False
+            self.enable_sdrf = False
+            self.msstats_input_valid = False
+            # TODO what if multiple are found??
+            for f in self.find_log_files("quantms/exp_design", filecontents=False):
+                self.exp_design = os.path.join(f["root"], f['fn'])
+                self.enable_exp = True
+            if self.enable_exp == False:
```

Ruff

Avoid equality comparisons to `False`; use `if not self.enable_exp:` for false checks

Replace with `not self.enable_exp`

(E712)